README.md

```diff
@@ -18,6 +22,8 @@ You can use really whatever method you want to get up and running, but I origina
 run -d --net=host -v </path/to/config>:/home/steam/frozen-flame-dedicated/FrozenFlame/Saved --name=frozen-flame-dedicated ghcr.io/ragingdave/frozen-flame-docker:main 
 ```
 
+You can use the docker compose in the folder if you want.
+
 **NOTE: you still have to create the Game.ini after running the server for the first time to enable things like passwords or other configurations. You can find those instructions here: https://github.com/DreamsideInteractive/FrozenFlameServer#configuration-file**
```

Comment:
It looks like from these changes, the Game.ini would get created automatically?

Reply:
I use a Game.ini simplified for the moment.
I didn't imagine an other use case for the moment. Update the Game.ini each new recreation ? Or keep the same until the user remove it ? I let you the choice.
The problem with the Game.ini, it can change and add some options (and we have to update all the docker image each time). I think the user may change it manually after the first start of the server.